modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/partials/navigation.hbs

```diff
@@ -111,6 +111,14 @@
                         {{/if}}
                     {{/each}}
                 </div>
+                <div id="tags-collapse" class="tags-collapse" align="center">
+                    <p class="small">
+                        <span id="tags-collapse-text">Show all tags&nbsp;&nbsp;</span>
```

Comment:
AFAIK we cannot use CSS space property here. It defines the size of the space between two words. Here we need 2 spaces after a certain word. I referred to https://github.com/wso2/product-greg/blob/master/modules/es-extensions/store/extensions/app/greg_store/themes/store/partials/header.hbs#L43 and many other places we have followed the same pattern.